tools/linter_lib/custom_check.py

```diff
@@ -576,6 +576,7 @@
             # We have URL template and Pygments language name as placeholders
             # in the below template which we don't want to be translatable.
             "web/templates/settings/playground_settings_admin.hbs",
+            "templates/zerver/slack_import.html",
```

Comment:
`exclude_line` would be better if it's just field or two.

Reply:
Changed to use `exclude_line`

`("templates/zerver/slack_import.html", 'placeholder="xoxb-…"'),`